lib/ansible/modules/network/iosxr/iosxr_config.py

```diff
@@ -168,11 +168,11 @@
 """
 
 RETURN = """
-updates:
+commands:
   description: The set of commands that will be pushed to the remote device
-  returned: Only when lines is specified.
+  returned: If there are commands to run against the host
```

Comment:
returned: Only when `lines` in the arg_spec is specified.

^this looks better description to imply use of `lines` for `commands` list to be returned

Reply:
It's not just `lines` that contribute to the commands to be ran against the host. The commands could come from a template in `src`. 

`if any((module.params['lines'], module.params['src'])):`

Reply:
right